### ISSUE TITLE
Conflict building with Firebase

### ISSUE THREAD

**Issue by grassick** [dpa99c/cordova-plugin-request-location-accuracy#8]
When building this with the firebase plugin, which requires google-services 9.0.0, I get the following error.

```
Found com.google.android.gms:play-services-location:+, but version 9.0.0 is needed for the google-services plugin.
:processDebugGoogleServices FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processDebugGoogleServices'.
> Please fix the version conflict either by updating the version of the google-services plugin (information about the latest version is available at https://bintray.com/android/android-tools/com.google.gms.google-services/) or updating the version of com.google.android.gms to 9.0.0.
```

This is related to: https://github.com/intercom/intercom-cordova/issues/58

I am forking with a change to specify the exact play services version, but would be thrilled to have a better solution, as would many other developers.

**Comment by dpa99c** [comment]
The problem here lies with the Firebase plugin - by [pinning a specific version of Firebase](https://github.com/fechanique/cordova-plugin-fcm/blob/master/src/android/FCMPlugin.gradle#L16) (and therefore the Google Play Services library), rather than the latest version (`+`), there is going to be a collision with any other Cordova plugin which specifies a differing version (or even the most recent version) of the Google Play Services library. There is nothing than can be done generically by this plugin to resolve that- if I change this plugin to fix the version at 9.0.0, then as soon as the Firebase plugin changes its fixed version of Firebase from 9.0.0, or another plugin specifies a different version,  it will break again.

FYI the latest version of Google Play Services library is v9.2, but also v9.0.2 resolves Firebase-related issues - [see here](https://developers.google.com/android/guides/releases#june_2016_-_v92)

For the above reasons, I will not modify this plugin to suit the whims of the Firebase plugin, but instead you should petition the author of the Firebase plugin to resolve this.

**Comment by grassick** [comment]
Good point. I'll create an issue there. Thanks for the details response!
